### PR TITLE
fix: ad slot position

### PR DESCRIPTION
**Pull request by parkjh**
jira: https://nidigitalsolutions.jira.com/browse/REPLAT-1407

fixes `inline-ad` for articles

### CODE REVIEW

File: packages/ad/src/utils/generate-config.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ const slotPositions = {
   "ad-pixelteads": 2,
   "article-sponsored-ad": 2,
   default: 0,
-  header: 1,
+  header: 0,
```
Comment (aronshamash) [inline packages/ad/src/utils/generate-config.js]: is this correct having `header` and `default` set to `0` ?

Comment (aronshamash) [reply packages/ad/src/utils/generate-config.js]: Is there a jira ticket for the bug ?

Comment (parkjh) [reply packages/ad/src/utils/generate-config.js]: @aronshamash I'm doing it as part of https://nidigitalsolutions.jira.com/browse/REPLAT-1407

Comment (aronshamash) [reply packages/ad/src/utils/generate-config.js]: thanks. Is it working now on staging ?

Comment (parkjh) [reply packages/ad/src/utils/generate-config.js]: @aronshamash what is working on staging?

Comment (aronshamash) [reply packages/ad/src/utils/generate-config.js]: @parkjh does the ad show?

Comment (parkjh) [reply packages/ad/src/utils/generate-config.js]: @aronshamash so it's only showing for the header but the error is that the slot name for the ad is called `inline-ad` even though it should be `header`... conversely the inline ad is empty as it does not have any ad slot id...

Comment (parkjh) [reply packages/ad/src/utils/generate-config.js]: so it doesn't really work on staging yet but I'm going to add a screenshot of my local render working correctly

**Pull request closed without merging**